utopia-remix/app/components/projectNotFound.css.ts

```diff
@@ -0,0 +1,7 @@
+import { style } from '@vanilla-extract/css'
+
+export const hover = style({
+  ':hover': {
+    backgroundColor: '#5DA9FF !important',
```

Comment:
is `important` needed? (just to double check so we don't have to chase them)

Reply:
unfotunately here yes, since the Radix classes override the vanilla-extract ones. I think this will be an issue that we'll need to solve in a more "strategic" level when we decide which design system we want to use and how we want to configure it

Reply:
😢